libs/shared-config/src/lib/config.ts

```diff
@@ -445,6 +443,16 @@ export const COURSES = [
 ] as const
 export type CourseKey = typeof COURSES[number]['id']
 
+export const HOW_DID_HEAR_ABOUT_REDI_OPTIONS = [
```

Comment:
I'm a bit worried about keys and values being the same in this, a dropdown field data list. It would require a data migration at the database level if we want to change the wording ever so slightly, and wouldn't work well with future localization. I did this for the languages list, but regret that choice.

Can we make this a key-value object like it's done mostly elsewhere?

Also, since this is (I assume) a TS-only list, I suggest we store it in the TP config file, not here in the shared config file. This reminds me, I'll make a #refactor note for myself to move some of the CON-only configs into a CON-exclusive config file.